tests/phpunit/tests/media.php

```diff
@@ -4015,6 +4142,19 @@ static function() use ( $threshold ) {
 			}
 		);
 	}
+
+	/**
+	 * Update global wp_query with new query.
+	 * And, reset content media count and remove omit loading attribute filters.
+	 */
+	public function get_new_wp_query_and_reset_variables() {
```

Comment:
See my comment above, bundling unrelated things in a single helper method isn't a good practice. I'd prefer if we avoided this method and used more granular helper methods (like the two last calls in this method are already).

Reply:
> 

@felixarntz I agree that bundling unrelated function would be a bad practice. I have gone ahead and created utils the way you proposed.